paddle/fluid/operators/alloc_continuous_space_op.cc

```diff
@@ -105,10 +108,10 @@ class AllocContinuousSpaceKernel : public framework::OpKernel<T> {
           .Resize(dim);
       len = Alignment(len * size_of_dtype, context.GetPlace()) / size_of_dtype;
       offset += len;
-      VLOG(10) << "alloc_space_for_vars: output(" << out_var_names[i]
-               << ") ,dim:(" << dim << ")"
-               << " Address: " << out_tensors[i]->data<void>();
+      ss << "output(" << out_var_names[i] << ")  dim:(" << dim << ")"
+         << " address: " << out_tensors[i]->data<void>() << ", ";
     }
+    VLOG(10) << ss.str();
```

Comment:
Why change the above code?
If VLOG is not open, the debug info doesn't need to print.

Reply:
It doesn't change the log level but the content. All the grouped vars will be printed in one line. It's more convenient